src/GafferSceneUI/CropWindowTool.cpp

```diff
@@ -459,7 +459,9 @@ class CropWindowTool::Rectangle : public GafferUI::Gadget
 			else
 			{
 				const ImageGadget *imageGadget = static_cast<const ImageGadget *>( viewportGadget->getPrimaryChild() );
-				return imageGadget->pixelAt( event.line );
+				V2f pixel = imageGadget->pixelAt( event.line );
+				pixel.x *= imageGadget->getImage()->format().getPixelAspect();
```

Comment:
Golden rule : when triggering a compute, make sure you are using a suitable context. In this case, that would be `ImageGadget::getContext()`.

Reply:
Doh! Sorry, yes https://github.com/GafferHQ/gaffer/pull/3691/commits/1bc10aa6f9efe2d43983cfffe5dda031e150037b.